modules/cli-options/src/main/scala/scala/cli/commands/SpecificationLevel.scala

```diff
@@ -0,0 +1,40 @@
+package scala.cli.commands
+
+import caseapp.annotation.Tag
+import os.copy
+
+sealed trait SpecificationLevel extends Product with Serializable {
+  def md = toString() + " have"
+}
+object SpecificationLevel {
+  case object MUST   extends SpecificationLevel
```

Comment:
I think some minimalistic scaladocs on what each tag stands for would be cool. I mean, `must` and `should` are rather self explanatory, but i.e. `implementation` much less so.